package.json

```diff
@@ -124,6 +124,7 @@
   "dependencies": {
     "@babel/polyfill": "^7.0.0",
     "auto-bind": "^5.0.1",
+    "base64url": "^3.0.1",
```

Comment:
Is this dependency really necessary? Could we also go for a combination of `atob` and `encodeURIComponent` (and the respective inverses)?